ravenframework/Optimizers/RavenSampled.py

```diff
@@ -312,57 +316,127 @@ def finalizeSampler(self, failedRuns):
       @ In, failedRuns, list, runs that failed as part of this sampling
       @ Out, None
     """
-    # get and print the best trajectory obtained
-    bestValue = None
-    bestTraj = None
-    bestPoint = None
-    s = -1 if self._minMax == 'max' else 1
-    # check converged trajectories
-    self.raiseAMessage('*' * 80)
-    self.raiseAMessage('Optimizer Final Results:')
-    self.raiseADebug('')
-    self.raiseADebug(' - Trajectory Results:')
-    self.raiseADebug('  TRAJ   STATUS    VALUE')
-    statusTemplate = '   {traj:2d}  {status:^11s}  {val: 1.3e}'
-    # print cancelled traj
-    for traj, info in self._cancelledTraj.items():
-      val = info['value']
-      status = info['reason']
-      self.raiseADebug(statusTemplate.format(status=status, traj=traj, val=s * val))
-    # check converged traj
-    for traj, info in self._convergedTraj.items():
+    if len(self._objectiveVar) == 1:
```

Comment:
@JunyungKim, in single-objective, this can still have a length > 1, for example, if the name is `ans` the length will be 3 (i.e., the number of characters in that string). I suggest you change this in the getInputSpecs to a list of strings, or use `type()` here as you have done in other places. P.S. I prefer the first solution.

Reply:
@JunyungKim, could you please push the XML input files? Also, the two new benchmark tests should be added to `tests` and the results to gold so that I can test my fix.